TradeItIosEmsApi/TradeItAuthLinkResult.h

```diff
@@ -11,8 +11,8 @@
 
 @interface TradeItAuthLinkResult : TradeItResult
 
-@property (nullable) NSString *userId;
-@property (nullable) NSString *userToken;
+@property (nonnull) NSString *userId;
+@property (nonnull) NSString *userToken;
```

Comment:
I don't know if I can change these - the JSON lib we are using is a bit opaque. Were these set to `nullable` just for safety? Do we ever change our behaviour if they are `nil`? Not that I saw.